pkg/monitortests/clusterversionoperator/legacycvomonitortests/operators.go

```diff
@@ -598,8 +598,190 @@ func testOperatorStateTransitions(events monitorapi.Intervals, conditionTypes []
 
 			if len(fatal) == 0 {
 				// add a success so we flake (or pass) and don't fail
-				ret = append(ret, &junitapi.JUnitTestCase{Name: testName})
+				ret = append(ret, &junitapi.JUnitTestCase{Name: testName, SystemOut: "Passing the case to make the overall test case flake as the previous failure is expected"})
```

Comment:
So you see the added message about flakes even if there is no exception above?

Reply:
Seems working.

https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/test-platform-results/logs/openshift-origin-30296-ci-4.21-upgrade-from-stable-4.20-e2e-gcp-ovn-rt-upgrade/1981554197551648768/artifacts/e2e-gcp-ovn-rt-upgrade/openshift-e2e-test/artifacts/junit/e2e-monitor-tests__20251024-042412.xml

The message is only on the flaky tests.